### ISSUE TITLE
Better folder structure and logic incapsulation

### ISSUE THREAD

**Issue by oleksbard** [ivanoff/gl#3]
Your routes+controllers+models are in main script:

``` js
app.get('/', function (req, res) { res.redirect('/users') })
```

Better to have something like this: 
- **/models** - contains all your ORM models
- **/views** - contains your view-templates _no need to have any in this hometask_
- **/public** - contains all static content _no need to have any in this hometask_
- **/controllers** - contain all your express routes, separated by module/area of your application

In other words: 
- create separate _routes.js_ module and move all your routes there
- from _routes.js_ module remove all route handlers and put them to the _/controllers_ (grouped by domain, you have one -> **users**)

Example:

``` js
function Routing(app){
  var UsersController = require('../controllers/users');

  app.get('/users', UsersController.getAll);
}

module.exports = Routing;
```

**Comment by ivanoff** [comment]
Посмотри, пожалуйста, изменения, есть ли замечания по структуре?

**Comment by oleksbard** [comment]
Take a look at this example: 

routing.js:

``` js
var usersController = require('./controllers/users');

module.exports = function(app){
  app.get('/users', usersController.getAll);
}
```

constrollers/usersController.js:

``` js
exports.getAll = function(){
  // get all users rom DB
}
```

app.js

``` js
var Express = require('express');
var app = new Express();
var routing = require('./routing');
routing(app);
```

Controllers doesn't know anything about routing and routes doesn't know anything about how they handled.

**Comment by ivanoff** [comment]
Did you mean, that code has to be looks like after that commit:
https://github.com/ivanoff/gl/commit/ce32b24f20bb4da0e40f67197d828f367085938e

**Comment by oleksbard** [comment]
Yep.
